client/notifications/index.jsx

```diff
@@ -159,6 +159,9 @@ export class Notifications extends Component {
 			OPEN_POST: [ ( store, { siteId, postId, href } ) => {
 				if ( config.isEnabled( 'notifications/link-to-reader' ) ) {
 					this.props.checkToggle();
+					analytics.tracks.recordEvent( 'calypso_notifications_open_post', {
+						siteId, postId
+					} );
```

Comment:
whoa! how about them Redux analytics actions dude! let's not reintroduce test-destroying imperative browser-demanding direct network calls here :smile: